### ISSUE TITLE
ModelViewer widget does not load after publishing on Google Play

### ISSUE THREAD

**Issue by roked** [drydart/model_viewer.dart#27]
Hello,

Recently I developed an app and under the assets folder I uploaded several models in the .glb format. While I am running the app on physical device (using Android Studio) everything works fine and the models load. As soon as I publish a release on google play when I navigate to the same page where the 3D model is expected to load, nothing happens and the screen stays white. 

What could be the reason for that?

I am running latest version of the plug in as well as latest stable version of flutter.

Thank you.

**Comment by uxduck** [comment]
Is @roked the same person that posted this: https://stackoverflow.com/questions/66992871/3d-model-viewer-not-loading?noredirect=1#comment118419997_66992871

In that case, this bug was caused because readme/ documentation for this repo doesn't include ` <uses-permission android:name="android.permission.INTERNET"/>` to the Android manifest. This is actually a confusing problem, because its automatically added to debug builds, and will only stop working for release. Most people won't face this since their app will get the permission for another reason: all apps want internet 😅

We should add something like this: https://docs.microsoft.com/en-us/xamarin/android/troubleshooting/questions/android-internet

**Comment by roked** [comment]
Yes, I am the owner of the StackOverflow thread. It's frustrating because you actually won't take that in account if you are doing an offline 3D app (just like me 😂).
